dockerfile/ubuntu-20.04-amd64.Dockerfile

```diff
@@ -44,6 +44,7 @@ RUN cd $TT_METAL_INFRA_DIR \
     && wget https://apt.llvm.org/llvm.sh \
     && chmod u+x llvm.sh \
     && ./llvm.sh 17
+    && apt install -y install libc++-17-dev libc++abi-17-dev
```

Comment:
do you think this would be better placed in the requirements.txt or requirements-dev.txt files under scripts/docker?

Reply:
You can't install `libc++-17` until you run `llvm.sh`, so I like these lines together.

Reply:
ok, sounds good then